src/olympia/addons/tests/test_views.py

```diff
@@ -1200,8 +1201,12 @@ class TestAddonSearchView(ESTestCase):
     def setUp(self):
         super().setUp()
         self.url = reverse_ns('addon-search')
+        # Create return to AMO waffle switches used for rta: guid search, then
+        # fetch them once to get them in the cache.
         self.create_switch('return-to-amo', active=True)
+        self.create_switch('return-to-amo-for-all-listed', active=False)
         switch_is_active('return-to-amo')
+        switch_is_active('return-to-amo-for-all-listed')
```

Comment:
heh, this `switch_is_active` statement fooled me - I thought it was an assert (that should have failed!)

Reply:
Yeah, that's why I added the comment. We don't really care about the return value, but we do need to trigger a check once so that the result is cached (avoiding database queries in the tests)